CriteoPublisherSdk/Sources/Context/CR_InternalContextProvider.m

```diff
@@ -177,4 +186,8 @@ - (void)setNonNullObject:(id)object
   return components;
 }
 
+- (CGSize)screenSize {
```

Comment:
Isn't the sceenSize and the orientation already available with DeviceUtil ?
AFAIR there is some tooling on DeviceUtil to make it testable.

Reply:
AFAIK, There is no `DeviceUtil`?
If you are talking about `DeviceInfo`, I don't see the added value of having such (complex) dependency for a simple case like this?

Reply:
The added value is to have a single tested/mockable entry point to compute such value.
I may be bias because of Android: there is really a lot of way to compute the screen size: with/without margin bar, rotation, ...

Reply:
In this particular case I doubt we want to compute anything, but want the hardware physical size only, don't we?